doc/developer/local_setup.md

```diff
@@ -28,15 +28,27 @@ Install all ruby and node dependencies (in the core folder):
 
 In the core directory:
 
-    ./bin/wagon activate generic
+    ./bin/active_wagon.rb generic
```

Comment:
we could rename `bin/active_wagon.rb` to `bin/activate_wagon`, right? with the extension it feels like a library. For any executable, the extension should not matter as the `#!/path/to/runtime` is present.

Reply:
Sure :+1: